### ISSUE TITLE
Settings window appears randomly when moving the mouse (I think).

### ISSUE THREAD

**Issue by fletom** [YePpHa/YouTubeCenter#1465]
I don't have a way of reliably reproducing this but once every few minutes while browsing YouTube the YouTubeCenter settings window appears randomly on the screen, without going anywhere near the settings button in the top right. It seems to happen when I'm moving the mouse around to do things.

Has anyone else experienced this?

I searched for similar issues but didn't see anything. But I could read through all 644 (!) open issues to check for sure. If this is a duplicate please close it and refer me to the main issue. Thanks!

**Comment by Eisys** [comment]
I keep up with the issues quite well and this is the first time I've seen something like this.
You should say what version of what browser you have, and the version of Youtube Center you have.

**Comment by fletom** [comment]
Right, sorry! Latest Safari (8.0.3) on OS X Yosemite, YouTubeCenter version 2.1.7.

**Comment by Eisys** [comment]
Ah well, I think here's the problem. 2.1.7 is really old by now and has lot of bugs.
I suggest installing the latest DEV version, which also has tons of new features:
https://github.com/YePpHa/YouTubeCenter/wiki/Developer-Version
Although, Safari is a less popular choice of browsers so I'm not sure if the DEV version is updated for it, but give it a try!

**Comment by fletom** [comment]
@Eisys YouTube 2.1.7 is the latest version you have published for safari here (https://github.com/YePpHa/YouTubeCenter/wiki#safari-extension) so it could hardly be "really old".

I've downloaded the dev version and I'll post back here if I encounter the issue again.

**Comment by YePpHa** [comment]
v2.1.7 is a couple of months old and YouTube have updated a lot since then. So it's possible that there are a few bugs in v2.1.7 caused by YouTube updates.
